dtfs-central-api/src/services/deal-cancellation/post-deal-cancellation.service.ts

```diff
@@ -0,0 +1 @@
+export const shouldDealBeCancelledImmediately = (effectiveFrom: number) => new Date().valueOf() >= effectiveFrom;
```

Comment:
I'm not sure how service layers should work - but I'm expecting probably they should have more than this? Any insights much appreciated